extractor.cpp

```diff
@@ -133,7 +133,7 @@ void Extractor::onFileListLoaded(const QStringList &files)
 		return;
 	}
 	emit extractionStarted(files.size());
-	while (!m_files.isEmpty() && m_activeFiles < MAX_ACTIVE_FILES) {
+	while (!m_files.isEmpty() && m_activeFiles < QThread::idealThreadCount()) {
```

Comment:
do you know what this method returns? Is it the total number of threads that the CPU has? Perhaps we could remove 1 or 2 from it to leave some available resources.

Reply:
Yes, it returns the total number of threads (or 1 if it can't detect for some reason):
https://doc.qt.io/qt-5/qthread.html#idealThreadCount

I think removing one might be sensible.